### PR TITLE
add dydx

### PR DESCRIPTION
**Pull request by dgogel**
adding dydx info

### CODE REVIEW

File: protocols/dydx/index.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "dydx",
+  "description": "DYDX is a governance token that allows holders to govern the dYdX Layer 2 perpetuals protocol.",
+  "path": "example",
```
Comment (Rashmi-278) [inline protocols/dydx/index.json:4]: Please change path to 'dydx'

File: protocols/dydx/index.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "type": ["snapshot", "Aave Governance V2"]
```
Comment (Rashmi-278) [inline protocols/dydx/index.json:5]: Can you please share the snapshot and aave-gov links in the comments of this PR ?

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]
Love the detailed overview !

File: protocols/dydx/index.json (hunk 2 above)
Comment (dgogel) [reply protocols/dydx/index.json:5]: https://snapshot.org/#/dydxgov.eth

The DydxGovernor contract was inspired by Aave. Minor changes have been made by dYdX. 
Governor is deployed at 0x7E9B1672616FF6D6629Ef2879419aaE79A9018D2 - https://etherscan.io/address/0x7e9b1672616ff6d6629ef2879419aae79a9018d2

**Review by Rashmi-278 — COMMENTED** [review]
https://snapshot.org/#/dydxgov.eth

The DydxGovernor contract was inspired by Aave. Minor changes have been made by dYdX.
Governor is deployed at 0x7E9B1672616FF6D6629Ef2879419aaE79A9018D2 - https://etherscan.io/address/0x7e9b1672616ff6d6629ef2879419aae79a9018d2

**Review by Rashmi-278 — APPROVED** [review]

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**